shared/types/field/myinfo/base.ts

```diff
@@ -0,0 +1,137 @@
+import {
```

Comment:
does this need to be in shared? if it's only used by the client we can just use this file there

Reply:
sure, will do! will relocate to `admin-form/create/builder-and-design`!

Reply:
relocated them to ^ and `~features/myinfo/`